src/helpers/model/measureValueFilters.ts

```diff
@@ -1,107 +1,36 @@
-// (C) 2019 GoodData Corporation
-import isEmpty = require("lodash/isEmpty");
-import invariant = require("invariant");
-import { AFM } from "@gooddata/typings";
+// (C) 2020 GoodData Corporation
```

Comment:
The model was changed to generate `VisualizationInput` instead of AFM structures